src/schemas/index.ts

```diff
@@ -0,0 +1,24 @@
+import * as z from "zod";
+
+export const LoginSchema = z.object({
+  email: z.string().email({
+    message: "Email is required",
+  }),
+  password: z.string().min(1, {
+    message: "Password is required",
+  }),
+});
+
+export const RegisterSchema = z.object({
+  email: z.string().email({
+    message: "Email is required",
+  }),
+  password: z
```

Comment:
just FYI i think we should make stronger validation for password including upper-case, special character etc. but lets leave it for future improvement.

Reply:
im aware of that, i just wanted to merge these changes asap so i can connect BE with FE and then proceed with validation